### PR TITLE
CI: xfail html test #36467

### PR DESCRIPTION
**Pull request by fangchenli**
Part of #36467

### CODE REVIEW

**Review by simonjayhawkins — APPROVED** [review]

**Comment by simonjayhawkins** [comment]
I'm fairly certain that i've seen this on the backports too. so can either backport this or merge to master and I could backport if I see it again.

**Comment by jreback** [comment]
are we sure this is not coming from something inside read_html? eg something we are doing

**Comment by simonjayhawkins** [comment]
> are we sure this is not coming from something inside read_html? eg something we are doing

I guess we will be keeping the issue open until we are sure, but the problem will be less visible of course.

**Comment by alimcmaster1** [comment]
> are we sure this is not coming from something inside read_html? eg something we are doing

Agree - I think I diagnosed the root cause here: https://github.com/pandas-dev/pandas/pull/36480 (doesn't actually solve the problem though)

File: pandas/tests/io/test_html.py [hunk 1]
```diff
@@ -121,6 +121,7 @@ def test_to_html_compat(self):
         res = self.read_html(out, attrs={"class": "dataframe"}, index_col=0)[0]
         tm.assert_frame_equal(res, df)
 
+    @pytest.mark.xfail(reason="ResourceWarning from html5lib #36467", strict=False)
```
Comment (jbrockmendel) [inline pandas/tests/io/test_html.py:124]: no way to make this strict?

**Comment by jreback** [comment]
yeah i think we should NOT suppress these as they tend to get ignored

File: pandas/tests/io/test_html.py (hunk 1 above)
Comment (VirosaLi) [reply pandas/tests/io/test_html.py:124]: Not sure how. Let me think abouit it.

**Comment by simonjayhawkins** [comment]
> yeah i think we should NOT suppress these as they tend to get ignored

agree since it appears issues can get closed and the xfails not removed. eg. #35660 (have opened #36524)

**Pull request closed without merging**